### DISCUSSION TITLE
Dúvida sobre commit recente

### DISCUSSION THREAD

**Discussion by AbelChiquetti** [RamonSilva20/mapos#2594] [General]
Olá, estou acompanhando as commits pra deixar o meu sistema atualizado, mas vi que 2 dias atrás foi feito uma grande atualização e lançaram uma versão nova. Eu atualizei os arquivos que foram modificados no meu, mas depois foi feito o revert dessa commit. Devo dar revert no meu também? Ou posso manter como está

**Comment by Pr3d4dor** [comment]
Precisa dar revert sim pois esse commit está com problema.
